recipes/pytest-redis/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pytest-redis" %}
+{% set version = "2.4.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/pytest-redis-{{ version }}.tar.gz
+  sha256: 8a07520abed3cd341e8da1793059aa5717b02e56c43e7c76435db682cede10aa
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - pytest >=6.2.0
+    - port-for >=0.6.0
+    - mirakuru
+    - redis-py
+
+test:
+  imports:
+    - pytest_redis
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/ClearcodeHQ/pytest-redis
+  summary: Redis fixtures and fixture factories for Pytest.
+  license: LGPL-3.0-only
```

Looks like it's [actually](https://github.com/ClearcodeHQ/pytest-redis/blob/c020d6b7d7f78da117f67252dc43f6f4755dc112/src/pytest_redis/__init__.py#L9-L10):
```suggestion
  license: LGPL-3.0-or-later
```